style.scss

```diff
@@ -0,0 +1,321 @@
+$mainColor: #ffe8da;
```

Comment:
Nebála bych se proměnné rozdělit/odkomentovat dle toho, čeho se týkají. :) Aby kdykoli přijdeš přidat barvu nebo font hned bylo jasné, kam ho do toho dlouhého seznamu dát. Ve velkých projektech bys třeba na barvy měla i samostatný soubor.

Reply:
Pro přehlednost určitě super, doplnila jsem. Děkuji :)